doc/user/events/widget.rst

```diff
@@ -329,125 +334,70 @@ Hosted or pretix Enterprise are active, you can pass the following fields:
 * If you use the campaigns plugin, you can pass a campaign ID as a value to ``data-campaign``. This way, all orders
   made through this widget will be counted towards this campaign.
 
-* If you use the tracking plugin, you can enable cross-domain tracking. To do so, you need to initialize the 
-  pretix-widget manually. Use the html code to embed the widget and add one the following code snippets. Make sure to
-  replace all occurrences of <MEASUREMENT_ID> with your Google Analytics MEASUREMENT_ID (UA-XXXXXXX-X or G-XXXXXXXX)
+* If you use the tracking plugin, you can enable cross-domain tracking. Use the HTML code to embed the widget and 
```

Comment:
Should we add a paragraph that they do not need cross-domain tracking when they run pretix on a subdomain of the main tracking domain? Because then it’s not cross-domain in at least Google Analytics’ terms of cross-domain.

Reply:
Yes, please!